testing/file_test/test_file.h

```diff
@@ -58,6 +58,12 @@ struct TestFile {
   // Files in the test, generated by content and splits.
   llvm::SmallVector<Split> file_splits;
 
+  // Paths to files to be included in the test file, and for each test split.
+  // What 'included' means is up to the `FileTestBase` implementation. For the
+  // toolchain/ they are added to the compile command line, making any package
+  // they export available to the test.
```

Comment:
With the changes, I think it's not really up to the individual implementation. Since they're added to args here, they're essentially behaving as parts of the file's test suite, aren't they?

Also, what does it mean by "for each test split"?

Maybe:

```suggestion
  // Paths to files to be included in the test. They are added to `test_args`
  // in `Run`, and also passed to `FileTestBase::SetIncludeFiles`.
```

Reply:
Done, but adjusted for the changes to the `fs`